### PR TITLE
Prize Ball Crate

### PR DESCRIPTION
**Pull request by ShatteredSwords**
# Description

Adds a prize ball crate as a replacement for the plushie crate. It's 10 for 5k. This is the equivalent of buying the bulk carnival plushies instead of going to the carnival. I hope you like gacha games!

---

# Changelog

:cl:
- add: Prize Ball Crates can now be ordered from the market

### CODE REVIEW

**Comment by ShatteredSwords** [comment]
No idea why build and test failed here, I have done nothing to the pet rock lol

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Sadly, Test Fail is valid and need to be resolved.

Pet Rock got a price/value if i remember, give it a look.

Tho looking at this may reaveal some clues:

"Attempted to spawn an entity with an invalid prototype: BasePetRock"

**Comment by ShatteredSwords** [comment]
Test fail invalid, smell my code you fops

File: Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml [hunk 1]
```diff
@@ -282,7 +282,7 @@
       - id: CrayonBox
         prob: 0.80
         orGroup: Prize
-      - id: BasePetRock
+      - id: PetRock
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Objects/Fun/prizeticket.yml:285]: Thank you!

**Review by FoxxoTrystan — APPROVED** [review]

**Review by Memeji — APPROVED** [review]

**Pull request merged**